sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/NestedColumnAliasing.scala

```diff
@@ -204,15 +226,8 @@ object GeneratorNestedColumnAliasing {
       g: Generate,
```

Comment:
What about removing this method and call `NestedColumnAliasing.replaceChildrenWithAliases` directly above?